apps/fhir/server/authentication.py

```diff
@@ -48,13 +71,25 @@ def search_fhir_id_by_identifier(search_identifier, request=None):
     # Add headers for FHIR backend logging, including auth_flow_dict
     if request:
         # Get auth flow session values.
+        # pull in the user who owns the app (organization)
         auth_flow_dict = get_session_auth_flow_trace(request)
-
         headers = generate_info_headers(request)
+        # may be part of the contract with BFD
         headers['BlueButton-AuthUuid'] = auth_flow_dict.get('auth_uuid', '')
         headers['BlueButton-AuthAppId'] = auth_flow_dict.get('auth_app_id', '')
         headers['BlueButton-AuthAppName'] = auth_flow_dict.get('auth_app_name', '')
         headers['BlueButton-AuthClientId'] = auth_flow_dict.get('auth_client_id', '')
+
+        app, user = get_app_and_org(auth_flow_dict.get('auth_client_id', None))
+
+        headers['auth_uuid'] = auth_flow_dict.get('auth_uuid', '')
+        headers['auth_app_id'] = auth_flow_dict.get('auth_app_id', '')
+        headers['auth_app_name'] = auth_flow_dict.get('auth_app_name', '')
+        headers['auth_client_id'] = auth_flow_dict.get('auth_client_id', '')
+        headers['auth_app_active'] = "true" if app and app.active else "false"
+        headers['auth_organization_id'] = str(user.id) if user else ""
+        headers['auth_organization_name'] = user.username if user else ""
+        headers['auth_organization_active'] = "true" if user and user.is_active else "false"
```

Comment:
@JFU-GIT This section related to updating the auth flow logging headers should be removed. Will work to add anything additional in BB2-280/181.